.github/workflows/chromatic.yml

```
@@ -0,0 +1,27 @@
+# .github/workflows/chromatic.yml
+
+# Workflow name
+name: 'Chromatic'
+
+# Event for the workflow
+on: push
```

Should this be on PR and pushes to main branches only?

```suggestion
on:
  push:
    branches:
    - main
    - release-*
  pull_request:
    branches:
    - main
    - release-*

concurrency:
  group: ${{ github.workflow }}-${{ github.event.number || github.ref }}
  cancel-in-progress: true
```

Also cancel previous runs if branch is updated?